src/providers/ldap/sdap_utils.c

```diff
@@ -206,6 +206,39 @@ char *sdap_combine_filters(TALLOC_CTX *mem_ctx,
     return sdap_combine_filters_ex(mem_ctx, '&', base_filter, extra_filter);
 }
 
+/* we check the principal and if it contains our realm, then we drop it
+ * for the comparison with sAMAccountName */
+
+char *principal_string_to_sAMAccountName(TALLOC_CTX *mem_ctx,
```

Comment:
switch to all lowercase `principal_string_to_samaccountname` to follow C naming convention